cmake/CMakeLists.txt

```diff
@@ -238,6 +254,12 @@ else()
     string(APPEND CMAKE_C_FLAGS_RELEASE " -march=native -mtune=native")
     string(APPEND CMAKE_CXX_FLAGS_RELWITHDEBINFO " -march=native -mtune=native")
     string(APPEND CMAKE_C_FLAGS_RELWITHDEBINFO " -march=native -mtune=native")
+  elseif  (onnxruntime_USE_AVX)
```

Comment:
nit: Why the different spacing style here relative to the above and surrounding code?